eox_nelp/api_clients/__init__.py

```diff
@@ -116,3 +117,36 @@ def make_get(self, path, payload):
             "error": True,
             "message": f"Invalid response with status {response.status_code}"
         }
+
+
+class AbstractSOAPClient(AbstractApiClient):
+    """This abstract class is an extension of AbstractApiClient that includes
+    a common POST method whose expected result is a xml response.
+    """
+
+    def make_post(self, path, data):
```

Comment:
Why do you select that the response of this method is a beautiful soup instance instead of a response.json??
I think in the rest Abstract Api client 
https://github.com/eduNEXT/eox-nelp/blob/e196a43694caf6a47f6e9fb8bf0c4de65b008627/eox_nelp/api_clients/__init__.py#L73-L76 the response is a "json" in this case dict. But comparing:
- rest -> json representation
- soap -> XML ??
So this method could return the XML string content instead of beatifoulsoup? I only ask for the reasons

Reply:
There is no a special reason, the string is also ok for me